lives-of-primitive-quakers/modernized/16-chapter-15.adoc

```diff
@@ -253,8 +253,7 @@ resist the insinuations that were used to proselyte me to that way,
 I became quickly defiled with the pollutions thereof; until it pleased God,
 in his rich love and mercy, to deliver me out of those snares,
 and to give me a clear understanding of the evil of that way.`"^
-footnote:["`Treatise on Universal Love.`"--Barclay's Works, vol.
-iii. p. 186.]
+footnote:[Treatise on Universal Love, from [.book-title]#Barclay's Works#, vol. iii. p. 186.]
```

Comment:
Leave this for now, I'll take a look at why it wasn't working without removing the `--`